collect_app/src/main/java/org/odk/collect/android/logic/FormController.java

```diff
@@ -812,7 +810,7 @@ public void setLanguage(String language) {
      */
     public FormEntryPrompt[] getQuestionPrompts() throws RuntimeException {
 
-        ArrayList<FormIndex> indicies = new ArrayList<FormIndex>();
+        ArrayList<FormIndex> indicies = new ArrayList<>();
```

Comment:
Spelling, and it’s a good practice to make the type general: `List` in this case.